cleanba/environments.py

```diff
@@ -194,6 +202,9 @@ def __init__(self, env: gym.vector.VectorEnv):
         self.observation_space = batch_space(self.single_observation_space, n=self.num_envs)
         self.single_action_space = env.single_action_space
         self.action_space = env.action_space
+        if remove_last_action:
```

Comment:
rather than modifying, I'd trust constructing a new Discrete action space more. What if there's state other than the `.n`? It's not the case but the code reader cannot easily tell.

```suggestion
        if remove_last_action:
            assert isinstance(self.single_action_space, gym.spaces.Discrete)
            self.single_action_space = gym.spaces.Discrete(self.single_action_space - 1)
            self.action_space = batch_space(self.single_action_space, n=self.num_envs) 
```